api/src/wcs_api/services/video_analysis/response_sanitizer.py

```
@@ -887,6 +887,93 @@ def _fmt_timestamp(sec: float) -> str:
+            break
+        quality = (p.get("quality") or "").lower()
+        if quality not in ("solid", "excellent"):
+            continue
+        notes = (p.get("notes") or "").strip()
+        if not notes:
+            continue
+        name = (p.get("name") or "").strip()
+        if not name:
+            continue
+        cnt = p.get("count")
+        label = f"{name.capitalize()} (×{cnt})" if cnt else name.capitalize()
```

Ruff (0.15.10)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)